CTP/conf/ha_repl.conf

```diff
@@ -22,10 +22,22 @@
 #WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE 
 #USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# The password for ssh connection.
-# The port for ssh connection.
-default.ssh.pwd=<ssh pwd>
-default.ssh.port=<ssh port>
+# The default password for all instance nodes to configure ssh connection.
+# default.ssh.pwd=<ssh pwd>
+# The default port for all instance nodes to configure ssh connection.
+# default.ssh.port=<ssh port>
+# Define default parameters all instance nodes to configure cubrid.conf
```

Comment:
as is:
+# Define default parameters all instance nodes to configure cubrid.conf

to be:
+# Define default parameters for all instance nodes to configure cubrid.conf

Please also revise shell and isolation.